alertmanager.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
We might want to add a comment here mentioning that manual edits will not be preserved once we start generating this file. See https://jira.percona.com/browse/PMM-5702.
It is on me.